From aa503a9f322c07284a11e142e530b7dc1f4beb6b Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Tue, 07 Feb 2023 11:35:32 +0200
Subject: [PATCH] [SW-121541] sensors: reduced exposure

Some sensors needs to be screened out for a better user experience.

Change-Id: Ia4d861fe7bc5250891cc79fc9d4de46893ce3ce1
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/sensors/hl_sensors.conf b/package/sensors/hl_sensors.conf
index a8be7e2..f0aae83 100644
--- a/package/sensors/hl_sensors.conf
+++ b/package/sensors/hl_sensors.conf
@@ -42,10 +42,10 @@
 label temp2  "On Board W"
 label temp3  "On Board NE"
 label temp4  "On Board E"
-label temp5  "On Chip SE"
-label temp6  "On Chip SW"
-label temp7  "On Chip NE"
-label temp8  "On Chip NW"
+ignore temp5
+ignore temp6
+ignore temp7
+ignore temp8
 label temp9  "HBM TS1-SE"
 label temp10 "HBM TS2-SW"
 label temp11 "HBM TS3-NE"
@@ -130,20 +130,20 @@
 
 chip "HL225-pci-*" "HL225B-pci-*"
 
-label temp1 "On Chip 0"
-label temp2 "On Chip 1"
-label temp3 "On Chip 2"
-label temp4 "On Chip 3"
+ignore temp1
+ignore temp2
+ignore temp3
+ignore temp4
 label temp5 "HBM 0 Temp"
 label temp6 "HBM 1 Temp"
 label temp7 "HBM 2 Temp"
 label temp8 "HBM 3 Temp"
 label temp9 "HBM 4 Temp"
 label temp10 "HBM 5 Temp"
-label temp11 "On Chip TD 0"
-label temp12 "On Chip TD 1"
-label temp13 "On Chip TD 2"
-label temp14 "On Chip TD 3"
+ignore temp11
+ignore temp12
+ignore temp13
+ignore temp14
 label temp15 "On Board 2 Top"
 label temp16 "On Board 01 Top"
 label temp17 "On Board 01 Bot"
@@ -154,48 +154,48 @@
 label temp22 "VRM2 Temp"
 
 #PVT voltage sensors are mapped between 0 and 63
-label in0 "On Chip 0 CH1"
-label in1 "On Chip 0 CH5"
-label in2 "On Chip 0 CH9"
-label in3 "On Chip 1 CH1"
-label in4 "On Chip 1 CH3"
-label in5 "On Chip 1 CH4"
-label in6 "On Chip 1 CH5"
-label in7 "On Chip 1 CH8"
-label in8 "On Chip 1 CH15"
-label in9 "On Chip 2 CH1"
-label in10 "On Chip 2 CH5"
-label in11 "On Chip 2 CH6"
-label in12 "On Chip 2 CH7"
-label in13 "On Chip 2 CH10"
-label in14 "On Chip 2 CH15"
-label in15 "On Chip 3 CH1"
-label in16 "On Chip 3 CH5"
-label in17 "On Chip 3 CH10"
-label in18 "On Chip 3 CH11"
+ignore in0
+ignore in1
+ignore in2
+ignore in3
+ignore in4
+ignore in5
+ignore in6
+ignore in7
+ignore in8
+ignore in9
+ignore in10
+ignore in11
+ignore in12
+ignore in13
+ignore in14
+ignore in15
+ignore in16
+ignore in17
+ignore in18
 label in19 "ADC 54V"
-label in20 "VRM1 Input"
-label in21 "VRM1 Output"
-label in22 "VRM2 Input"
-label in23 "VRM2 VDD Sense"
-label in24 "VRM2 HBM"
-label in25 "CPLD VMON1"
-label in26 "CPLD VMON2"
-label in27 "CPLD VMON3"
-label in28 "CPLD VMON5"
-label in29 "CPLD VMON6"
-label in30 "CPLD VMON7"
-label in31 "CPLD VMON8"
-label in32 "CPLD VMON9"
+ignore in20
+ignore in21
+ignore in22
+ignore in23
+ignore in24
+ignore in25
+ignore in26
+ignore in27
+ignore in28
+ignore in29
+ignore in30
+ignore in31
+ignore in32
 label in33 "CPLD HIMON"
 
 label curr1 "ADC 54V Current"
 label curr2 "ADC 12V1 Current"
-label curr3 "VRM1 Input Current"
-label curr4 "VRM1 Output Current"
-label curr5 "VRM2 13.5V Current"
-label curr6 "VRM2 VDD Current"
-label curr7 "VRM2 HBM Current"
+ignore curr3
+ignore curr4
+ignore curr5
+ignore curr6
+ignore curr7
 
 label power1 "Power Draw"
 
